### ISSUE TITLE
Debug output in `?directread` client claims cache requests

### ISSUE THREAD

**Issue by jhiemstrawisc** [PelicanPlatform/pelican#1475]
This came up during the HTC24 tutorial. When the following was run:
```
pelican object get pelican:// ... /foo?directread -d
```
The debug output called the supplied origin a "cache". Since the client knows the returned link should be to an origin (because of the ?directread query) it should change the label.

**Comment by brianaydemir** [comment]
@jhiemstrawisc Did you happen to fix this in https://github.com/PelicanPlatform/pelican/commit/821a320b404041456c5753f1a6c47e9a6ede4c28, maybe somewhere around [here](https://github.com/PelicanPlatform/pelican/blame/db96b8bb6358643b7e532716d77c3d2b45416c82/client/handle_http.go#L1290)?

The only occurrence of "cache" I'm currently seeing in the debugging output is

```
time="2024-12-05T09:31:27-06:00" level=debug msg="Using cached federation info for https://osg-htc.org"
```

which I believe is accurate.

For reference:

```
[baydemir@local Desktop]> pelican --version
Version: 7.11.3
Build Date: 2024-11-12T22:02:19Z
Build Commit: b3df1c8757a5232d23bf12bed51287a72edc55a1
Built By: goreleaser
[baydemir@local Desktop]> pelican object get --debug 'pelican://osg-htc.org/nsdf/testing/stress/v1/1?directread' test.obj 1>get.log 2>&1
```
And the resulting log file: [get.log](https://github.com/user-attachments/files/18025719/get.log)

**Comment by jhiemstrawisc** [comment]
Astute observation! Looks like the issue has been resolved for some time 😃

**Comment by jhiemstrawisc** [comment]
Closed by #1543
